ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/ui/TemplateRenderer.java

```diff
@@ -14,11 +18,12 @@ public TemplateRenderer(MustacheFactory mf) {
     this.mf = mf;
   }
 
-  public String render(String name, Object scope) {
-
+  public String render(String name, Map<String, Object> scope, Locale locale) {
     var template = mf.compile(name);
     var w = new StringWriter();
-    template.execute(w, scope);
+    var modifiableScope = new HashMap<>(scope);
```

Comment:
I can't use Map.copyOf() because that will return an unmodifiableMap as well and I do need that to be modifiable.